src/app/auth/auth.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="form">
+  <div class="col-xs-12 col-md-6 col-md-offset-3">
+    <form>
+      <div class="form-group">
+        <label for="email">Email</label>
+        <input
+          type="email"
+          id="email"
+          class="form-control"
+          #userEmail
+          required
+        />
+      </div>
+      <div class="form-group">
+        <label for="password">Password</label>
+        <input
+          type="password"
+          id="password"
+          class="form-control"
+          #userPassword
+          required
+        />
+      </div>
+      <div>
+        <button
+          class="btn"
+          (click)="
```

Comment:
use template literals

Reply:
I tried to use template literals. But it isn't working in html.